test/ch/imedias/rsccfx/model/RsccTest.java

```diff
@@ -149,4 +149,41 @@ public void testValidateKey() {
       assertTrue(model.validateKey(validKey));
     }
   }
+
+  /**
+   * Test for {@link Rscc#startVncServer()}.
+   */
+  @Test
+  public void testStartVncServer() {
```

Comment:
this doesn't cover all cases yet with different scenarios like view only etc., but we can add them later if you want... maybe we could put a TODO here then?